kolibri/plugins/perseus_viewer/buildPerseus.js

```diff
@@ -0,0 +1,119 @@
+const fs = require('fs');
+const path = require('path');
+const execSync = require('child_process').execSync;
+const mkdirp = require('mkdirp');
+const rimraf = require('rimraf');
+const webpack = require('webpack');
+const buildConfig = require('./perseusBuildConfig');
+const extractPerseusMessages = require('./extractPerseusMessages');
+
+const currentCwd = process.cwd();
+
+const submodulesPath = path.resolve(__dirname, './submodules');
+
+rimraf.sync(submodulesPath);
+
+mkdirp.sync(submodulesPath);
+
+process.chdir(submodulesPath);
+
+console.log('Cloning the Learning Equality Perseus repo');
+
+execSync('git clone https://github.com/learningequality/perseus.git');
+
+console.log('Learning Equality Perseus repo successfully cloned');
+
+process.chdir('perseus');
+
+console.log('Fetching all perseus submodules');
+
+execSync('git submodule update --init --recursive');
+
+console.log('All perseus submodules fetched');
+
+console.log('Installing perseus dependencies');
+
+execSync('yarn install');
+
+console.log('Perseus dependencies installed');
+
+process.chdir(__dirname);
+
+const target = path.resolve(__dirname, './assets/dist');
+
+console.log('Clearing dist folder');
+
+rimraf.sync(target);
+
+mkdirp.sync(target);
+
+const cssPathRegex = /(url\(['"]?)([^"')]+)?(['"]?\),? ?)/g;
+
+const files = [
+  path.resolve(__dirname, './submodules/perseus/stylesheets/local-only/khan-exercise.css'),
+  path.resolve(__dirname, './submodules/perseus/lib/katex/katex.css'),
+  path.resolve(__dirname, './submodules/perseus/build/perseus.css'),
+  path.resolve(__dirname, './submodules/perseus/lib/mathquill/mathquill.css'),
+];
+
+const compiler = webpack(buildConfig);
+
+console.log('Creating Perseus bundle with webpack');
+
+const tokenStackRegex = /_token_stack:\n/g;
+
+const README = `The files in this directory are generated using the buildPerseus script inside this plugin.
+They should be regenerated using the same command in the unlikely event that Perseus ever needs
+to be updated.`;
```

Comment:
Maybe provide a link here to the section of the developer docs which describe how to do this? ;-)

But more seriously, the why what and how should get some documentation either in comments in this file or a mention in the dev docs.